### PR TITLE
SpinButton: Fix label font size regression and improve disabled state style

### PR DESCRIPTION
**Pull request by montselozanod**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes
A recent regression was making the font size of labels of a SpinButton  smaller by default then they originally were.

Besides that, the disabled state of the spinbutton did not render properly the arrow buttons nor grayed out the label.

##### Before change:
screen shot 2017-10-11 at 3 41 56 pm

##### After change:
screen shot 2017-10-11 at 3 14 39 pm

### CODE REVIEW

**Comment by lynamemi** [comment]
Are we sure the label needs to be greyed out? Checking with design

**Comment by lynamemi** [comment]
Confirmed, greyed out is right!

**Comment by christiango** [comment]
Thanks @lynamemi for confirming!

File: packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const getArrowButtonStyles = memoizeFunction((
       }
     },
     rootDisabled: {
-      opacity: 0,
+      opacity: 0.5,
       [MS_HIGHCONTRAST_ACTIVE]: {
         color: 'GrayText'
```
Comment (jspurlin) [inline packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts]: in high contrast we need to put the opacity back to 1

Comment (montselozanod) [reply packages/office-ui-fabric-react/src/components/SpinButton/SpinButton.styles.ts]: I'll update PR with this before you guys merge.

**Pull request merged**